### PR TITLE
Update: Feature flag export

### PR DESCRIPTION
**Pull request by ldkge**
Resolves #

## Description
Put feature flag behind a feature flag

## Proposed Changes

-

## Screenshots
Screen Shot 2020-12-08 at 5 00 26 PM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by cythrawll — APPROVED** [review]

**Review by kevinhinterlong — APPROVED** [review]

File: packages/reports/tests/dummy/config/environment.js [hunk 1]
```diff
@@ -48,7 +48,8 @@ module.exports = function(environment) {
         multipleExportFileTypes: ['pdf', 'png'],
         enabledNotifyIfData: true,
         enableContains: true,
-        enableTableEditing: true
+        enableTableEditing: true,
+        enableExport: false
```
Comment (jkusa) [inline packages/reports/tests/dummy/config/environment.js:52]: Do we need `enableExport` and `multipleExportFileTypes`?

Could we just have `exportFileTypes` and if it is empty or undefined export would be disabled?

File: packages/dashboards/tests/dummy/config/environment.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ module.exports = function(environment) {
       ],
       FEATURES: {
         enableDashboardExport: true,
-        multipleExportFileTypes: ['pdf', 'png'],
+        exportFileTypes: ['pdf', 'png'],
```
Comment (ldkge) [inline packages/dashboards/tests/dummy/config/environment.js:34]: It should be removed from dashboards as well right? @jkusa

**Comment by cythrawll** [comment]
Now that we go by that. should we add csv to be functional off that list rather than it being default?

File: packages/dashboards/tests/dummy/config/environment.js (hunk 2 above)
Comment (jkusa) [reply packages/dashboards/tests/dummy/config/environment.js:34]: yes

**Comment by jkusa** [comment]
> Now that we go by that. should we add csv to be functional off that list rather than it being default?

Yes

**Comment by ldkge** [comment]
Made the change for csv to be in that list instead of being default.

**Review by cythrawll — APPROVED** [review]

File: packages/dashboards/tests/dummy/config/environment.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ module.exports = function(environment) {
       ],
       FEATURES: {
         enableDashboardExport: true,
-        multipleExportFileTypes: ['pdf', 'png'],
+        exportFileTypes: [],
```
Comment (jkusa) [inline packages/dashboards/tests/dummy/config/environment.js:34]:
```suggestion
        exportFileTypes: ['csv', 'pdf', 'png'],
```

**Review by jkusa — APPROVED** [review]

**Pull request merged**